sprout/appliances/models.py

```diff
@@ -21,6 +21,7 @@
 from django.db.models.signals import pre_save
 from django.dispatch import receiver
 from django.utils import timezone
+from json_field import JSONField
```

Comment:
I don't see this in requirements change, besides, Django has its own implementation of JSONField afaik.

Reply:
It is present in Trackerbot. so this is just copy/paste.

Reply:
tested it locally. it can be used with sqlite. whereas default one spports only postgre.